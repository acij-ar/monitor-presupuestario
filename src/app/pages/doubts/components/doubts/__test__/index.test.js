const React = require('react');
const { shallow } = require('enzyme');

const Doubts = require('..');

describe('Doubts component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Doubts/>);
    expect(wrapper).toMatchSnapshot();
  });
});
