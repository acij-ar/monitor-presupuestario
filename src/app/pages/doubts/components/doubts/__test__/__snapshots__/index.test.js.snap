// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Doubts component should match snapshot 1`] = `
<Fragment>
  <Doubt
    checked={true}
    title="¿Pregunta 1?"
  >
    Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 2?"
  >
    Aliquam pellentesque mauris venenatis lacus auctor blandit. Praesent maximus ipsum nec enim pharetra sollicitudin in a nisl. Mauris malesuada vulputate efficitur. Proin fringilla enim in dignissim rhoncus. Ut semper neque leo, vitae faucibus dui dapibus id. Phasellus hendrerit magna ligula, quis efficitur ligula molestie vel. Nulla iaculis eget enim a hendrerit. Vivamus interdum, augue a congue tincidunt, ligula massa elementum erat, quis tristique eros purus ac lacus. Phasellus mollis pretium interdum. In pellentesque arcu non tortor venenatis, ac tristique mi lobortis. Morbi quam orci, convallis quis erat et, placerat bibendum lorem. Morbi dapibus elit in elit maximus, at mollis mauris cursus.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 3?"
  >
    The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 4?"
  >
    It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 5?"
  >
    There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet. It uses a dictionary of over 200 Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable. The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 6?"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas mi elit, tincidunt at est quis, viverra mattis massa. Fusce nisi mauris, blandit vestibulum finibus sit amet, hendrerit semper massa. Morbi auctor eu sapien rhoncus consectetur. Quisque consequat non purus et viverra. Aliquam auctor risus lectus, id pulvinar ipsum facilisis ut. Praesent congue varius urna at sollicitudin. Integer vitae fringilla risus, sed pulvinar ante. Phasellus porta maximus tellus vel fermentum. Sed non felis ac quam pretium vulputate. Suspendisse dapibus quam quis dui tempus cursus. Nunc in pretium sapien. Curabitur at sem nisl
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 7?"
  >
    Suspendisse mattis nisi ultricies est cursus, ut pulvinar tellus ultricies. Pellentesque sollicitudin blandit velit, ac facilisis nisi commodo in. Nulla vel fringilla dolor, at vehicula urna. Duis sed metus augue. Sed nec massa nunc.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 8?"
  >
    Mauris blandit lectus mauris, vel sollicitudin ante vestibulum at. Sed elementum dolor nulla. Vivamus nunc ex, bibendum nec sem eu, vehicula posuere felis. Curabitur in sapien sit amet libero condimentum consectetur. Aliquam sed neque mollis, sagittis nunc ornare, accumsan libero. Maecenas sit amet mi sapien. Nunc ac volutpat ligula. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque fringilla ante nec purus ullamcorper, nec molestie lectus egestas. Praesent lobortis blandit tempor. Donec elementum mauris eu orci gravida laoreet.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 9?"
  >
    In vulputate nisl ac sodales efficitur. Donec iaculis nulla ac urna sollicitudin imperdiet. Donec mollis magna egestas lacus sollicitudin ornare. Maecenas imperdiet luctus justo, id semper tortor vulputate eget. Sed eu massa ultricies, porttitor enim ac, tempor lorem. Praesent et tellus eget orci interdum iaculis. Donec at dui maximus, fringilla leo at, malesuada ligula. Ut in turpis fermentum, egestas elit ac, semper urna. Sed quis ex eget odio tincidunt hendrerit.
  </Doubt>
  <Doubt
    checked={false}
    title="¿Pregunta 10?"
  >
    Aliquam pellentesque mauris venenatis lacus auctor blandit. Praesent maximus ipsum nec enim pharetra sollicitudin in a nisl. Mauris malesuada vulputate efficitur. Proin fringilla enim in dignissim rhoncus. Ut semper neque leo, vitae faucibus dui dapibus id. Phasellus hendrerit magna ligula, quis efficitur ligula molestie vel. Nulla iaculis eget enim a hendrerit. Vivamus interdum, augue a congue tincidunt, ligula massa elementum erat, quis tristique eros purus ac lacus. Phasellus mollis pretium interdum. In pellentesque arcu non tortor venenatis, ac tristique mi lobortis. Morbi quam orci, convallis quis erat et, placerat bibendum lorem. Morbi dapibus elit in elit maximus, at mollis mauris cursus.
  </Doubt>
</Fragment>
`;
